src/plugins/visualizations/public/expressions/visualization_function.ts

```diff
@@ -94,6 +100,9 @@ export const visualization = (): ExpressionFunctionVisualization => ({
     const uiStateParams = args.uiState ? JSON.parse(args.uiState) : {};
     const uiState = new PersistedState(uiStateParams);
 
+    const aggConfigsState = args.aggConfigs ? JSON.parse(args.aggConfigs) : {};
```

Comment:
I don't think you want `{}` as a fallback here... `createAggConfigs` expects an array of serialized agg configs as the `aggConfigsState`:
```ts
createAggConfigs(
  indexPattern: IndexPattern,
  configStates: AggConfigSerialized[]
) => IAggConfigs;
```

So you can probably either fall back to `[]`, or to `undefined` (in which case `createAggConfigs` will default to `[]` anyway).

Reply:
You're right, I'll put `[]` here.
Unless you prefer `undefined`?

Reply:
Makes no difference IMHO. Inside `createAggConfigs` it will be `[]` either way